commons-jcs-jcache/src/main/java/org/apache/commons/jcs3/jcache/JCSCachingManager.java

```diff
@@ -189,8 +189,8 @@ private void assertNotClosed()
         assertNotClosed();
         assertNotNull(cacheName, "cacheName");
         assertNotNull(configuration, "configuration");
-        final Class<?> keyType = configuration == null ? Object.class : configuration.getKeyType();
```

Comment:
didnt check them all but this one was a typo configuration should check configuration.getKeyType() (resp value) to default to Object so instead of dropping it I would fix it, wdyt?

Reply:
Hi @rmannibucau 
Can you be more specific. Keep in mind i'm not an expert in JCS and still i´am learning about the logic of the project..
TY

Reply:
Expected pattern was `final var foo = a == null ? defaultvalue : a`. Here the default is object class so the configuration == null should be replaced by configuration.getKeyType() or configuration.getValueType() I think.